manifests/mod/info.pp

```diff
@@ -6,7 +6,22 @@
 ){
   include ::apache
   $_apache_version = pick($apache_version, $apache::apache_version)
-  apache::mod { 'info': }
+
+  if $::osfamily == 'Suse' {
+    if defined(Class['::apache::mod::worker']){
+      $suse_path = '/usr/lib64/apache2-worker'
+    } elsif defined(Class['::apache::mod::prefork']){
+      $suse_path = '/usr/lib64/apache2-prefork'
+    } else {
+      $suse_path = pick('/usr/lib64/apache2-prefork', '/usr/lib64/apache2-worker')
```

Comment:
Also, since prefork would be the default case then you don't need the `elsif` above.